### ISSUE TITLE
How to add space when insert?

### ISSUE THREAD

**Issue by msyesyan** [terryma/vim-multiple-cursors#133]
For example, when I select two words, like 'abc', 'abc', first I press v, and press i to enter an insert mode, then I input `ab cd`,but I cannot input the space character. By the way, when I input `b`, or `k`, or something else, it will raise an error like:
```
Error detected while processing function <SNR>107_process_user_input
```
and
```
E488: Trailing characters
```
What's wrong with my vim env? Thanks very much

**Comment by balta2ar** [comment]
I'd suggest to isolate the source of the problem first. Disable all other plugins, minimize your `~/.vimrc` and try it again. There might be some conflict with another plugin (it happened to me before) or a keybinding (use `:map` to see all your keybindings).

**Comment by msyesyan** [comment]
![cannot-insert-b-error](https://cloud.githubusercontent.com/assets/2162116/7128737/1fa8e41a-e289-11e4-9357-35f039f51998.gif)

**Comment by msyesyan** [comment]
please look at this gif, I just use the very original vimrc file which was generated by the NeoBundle, and I have searched all my map keys, it seems just ok, so can you help me?

**Comment by msyesyan** [comment]
thanks, and I have install the `kristijanhusak/vim-multiple-cursors` version, and it works now

**Comment by kristijanhusak** [comment]
@msyesyan this repo is synced with my fork several months ago, and not updated since then. So this should work properly here. Maybe some later PR broke it.

**Comment by msyesyan** [comment]
@kristijanhusak yes, I think so, I will maintain attention on it, thanks very much

**Comment by faceleg** [comment]
@msyesyan any progress?

**Comment by guyfedwards** [comment]
I am now seeing this issue, come about within the last 2 weeks for me

**Comment by garbelini** [comment]
Went back until it started working again and it looks like this is the PR that introduced the issue:
https://github.com/terryma/vim-multiple-cursors/pull/110

**Comment by balta2ar** [comment]
@garbelini @guyfedwards @msyesyan Please provide a reproducible minimal vimrc.

**Comment by guyfedwards** [comment]
@balta2ar I solved this by updating my vim to 7.4, cheers
